sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_access_control_client.py

```diff
@@ -0,0 +1,112 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from typing import TYPE_CHECKING
+
+from azure.core.tracing.decorator import distributed_trace
+
+from ._models import RoleAssignment, RoleDefinition
+from ._internal import KeyVaultClientBase
+
+if TYPE_CHECKING:
+    from typing import Any, Iterable
+
+
+class KeyVaultAccessControlClient(KeyVaultClientBase):
+    """Manages role-based access to Azure Key Vault.
+
+    :param str vault_url: URL of the vault the client will manage. This is also called the vault's "DNS Name".
+    :param credential: an object which can provide an access token for the vault, such as a credential from
+        :mod:`azure.identity`
+    """
+
+    # pylint:disable=protected-access
+
+    @distributed_trace
+    def create_role_assignment(self, role_scope, role_assignment_name, role_definition_id, principal_id, **kwargs):
+        # type: (str, str, str, str, **Any) -> RoleAssignment
+        """Create a role assignment.
+
+        :param str role_scope: scope the role assignment will apply over
+        :param role_assignment_name: a name for the role assignment. Must be a UUID.
```

Comment:
nit: the type hint should also allow uuid.UUID

Reply:
Thank you, have fixed.